api/controllers/PostController.js

```diff
@@ -58,10 +58,6 @@ const findTagId = req =>
   req.param('tag') && Tag.find(req.param('tag')).then(t => t.id)
 
 const queryForCommunity = function (req, res) {
-  if (TokenAuth.isAuthenticated(res)) {
-    if (!RequestValidation.requireTimeRange(req, res)) return
```

Comment:
@levity it's ok to take this out right? it's part of the specific use case with harlan?